Apsitvarkom.UnitTests/Models/Mapping/CleaningEventMappingTests.cs

```diff
@@ -23,7 +23,18 @@ public void OneTimeSetUp()
     }
 
     #region Request mappings
+    [Test]
+    public void ObjectIdentifyRequestToCleaningEventResponse()
```

Comment:
```suggestion
    public void ObjectIdentifyRequestToCleaningEvent()
```
we are mapping not to `CleaningEventResponse` here